libs/dot-primeng-theme-styles/src/scss/dotcms-theme/components/_datatable.scss

```diff
@@ -100,6 +100,7 @@ $datatable-bg-hover: rgba($brand-secondary_rgb, 0.05);
 .p-datatable .p-datatable-tbody > tr {
     background: $white;
     color: $black;
+    height: 4rem;
```

Comment:
We can have height because if we have to have 2 lines it will break.

Reply:
The reason why I used `height` instead of `min-height` is because that a table cell will always stretch if necessary, so height actually is the equivalent of min-height for a table cell.